tox.ini

```diff
@@ -23,6 +23,8 @@ commands =
     python -m pytest --cov-report term --cov=merlin -rxs tests/unit
 
 [testenv:test-gpu]
+passenv =
+    NR_USER
```

Comment:
Tox blocks all environment variables by default. We need to pass this in order to skip [this test that uses horovod](https://github.com/NVIDIA-Merlin/dataloader/blob/main/tests/unit/loader/test_tf_dataloader.py#L476-L478). The test was skipped previously because horovod was not installed in the ci runner image, but horovod was recently installed in the ci runner image because of multi-gpu support in Models.